libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -114,6 +147,133 @@ describe('ModalHelper', () => {
         });
       });
 
+      describe('sizing', () => {
+        it('modal should be default sized (medium), if size is not provided', async () => {
+          overlay = await modalHelper.showModalWindow({
+            title: 'Modal',
+            component: InputEmbeddedComponent,
+          });
+          ionModal = await ionModalController.getTop();
+          expect(ionModal).toBeTruthy();
+          expect(ionModal.classList.contains('small')).toBeFalse();
+          expect(ionModal.classList.contains('medium')).toBeTrue();
+          expect(ionModal.classList.contains('large')).toBeFalse();
+          await overlay.dismiss();
```

Comment:
Consider moving this to `afterEach` to ensure modal is dismissed if expect throws an error